src/augmented_dickey_fuller.jl

```diff
@@ -215,4 +216,4 @@ function show_params(io::IO, x::ADFTest, ident)
     println(io, ident, "Critical values at 1%, 5%, and 10%: ", x.cv')
 end
 
-pvalue(x::ADFTest) = HypothesisTests.pvalue(Normal(0, 1), adf_pv_aux(x.stat, x.deterministic); tail=:left)
+pvalue(x::ADFTest; tail=default_tail(x)) = HypothesisTests.pvalue(Normal(0, 1), adf_pv_aux(x.stat, x.deterministic); tail=tail)
```

Comment:
I don't understand why you don't use `tail=:left` here, since you already know the result of `default_tail` for that test. Same below. Also, 92 char limit.